pages/initiatives/gsoc/gsoc2022.md

```diff
@@ -51,7 +51,7 @@ their choice.
 Are you a student and want to code for an OWASP project? Here are the
 steps and some tips on getting started:
 
-1. Think of a good idea – For reference from last year see [GSoc {{page.year}} Ideas](gsoc{{page.year}}ideas).
+1. Think of a good idea – For reference from last year see [GSoc 2021 Ideas](gsoc2021ideas).
```

Comment:
This should use page.year -1 (dunno exact syntax off the top of my head)

Reply:
Oh, I came to this page from google search results and found this odd. I checked the [gsoc2021ideas.md](https://github.com/OWASP/www-community/blob/master/pages/initiatives/gsoc/gsoc2021ideas.md) and submitted the PR according to that. Shall I close this now?